src/shared/styles/custom-medias.css

```diff
@@ -0,0 +1,14 @@
+/* ==========================================================================
+   Custom media queries
+
+   Use it like so: @media (--layout-small) { ... }
+   ========================================================================== */
+
+/* Use these if you developing desktop first and want to target lower resolutions */
+
+@custom-media --layout-lte-small-mobile (width <= 20em); /* 320px * 0.0625 */
```

Comment:
I believe `375px` is a better breakpoint to target smaller devices and not the "normal" ones, like iPhone 6S.

Reply:
I believe that `320px` was the "problem" in most projects we work in. So I think that this should be a breakpoint to target small phones like `iPhone SE`. `375px` is the viewport width of an `iPhone X` and I think that doesn't represent "small" devices.

Reply:
-- this should also be part of the design system